From 848e5cd33de854b56fc898cb7d861d4ee1d0ab5e Mon Sep 17 00:00:00 2001
Message-ID: <848e5cd33de854b56fc898cb7d861d4ee1d0ab5e.1704423368.git.sharlatanus@gmail.com>
In-Reply-To: <cover.1704423368.git.sharlatanus@gmail.com>
References: <cover.1704423368.git.sharlatanus@gmail.com>
From: Sharlatan Hellseher <sharlatanus@gmail.com>
Date: Fri, 5 Jan 2024 02:44:29 +0000
Subject: [PATCH 2/3] gnu: yggdrasil: Sort inputs alphabetically.

* gnu/packages/networking.scm (yggdrasil)[propagated-inputs]: Sort
alphabetically to ease maintenance.

Change-Id: I336307433842bd1dce0cfb62d93be767e0a92dbb
---
 gnu/packages/networking.scm | 40 ++++++++++++++++++-------------------
 1 file changed, 20 insertions(+), 20 deletions(-)

diff --git a/gnu/packages/networking.scm b/gnu/packages/networking.scm
index c3796f7328..bf72fd7de2 100644
--- a/gnu/packages/networking.scm
+++ b/gnu/packages/networking.scm
@@ -4450,31 +4450,31 @@ (define-public yggdrasil
                #:deep? #true)))
        (cons go-golang-org-x-sys-0.8
              (map p
-                  (list go-golang-zx2c4-com-wireguard
-                        go-golang-org-x-text
-                        go-golang-org-x-net
-                        go-golang-org-x-crypto
-                        go-golang-org-x-tools
-                        go-netns
-                        go-netlink
+                  (list go-github-com-arceliar-ironwood
+                        go-github-com-arceliar-phony
                         go-github-com-bits-and-blooms-bitset
                         go-github-com-bits-and-blooms-bloom
-                        go-github-com-quic-go-quic-go
+                        go-github-com-cheggaaa-pb-v3
+                        go-github-com-fatih-color
+                        go-github-com-gologme-log
+                        go-github-com-hashicorp-go-syslog
                         go-github-com-hjson-hjson-go
-                        go-github-com-olekukonko-tablewriter
-                        go-github-com-mitchellh-mapstructure
-                        go-github-com-mattn-go-runewidth
-                        go-github-com-mattn-go-isatty
-                        go-github-com-mattn-go-colorable
-                        go-github-com-kardianos-minwinsvc
                         go-github-com-hjson-hjson-go
-                        go-github-com-hashicorp-go-syslog
-                        go-github-com-gologme-log
-                        go-github-com-fatih-color
-                        go-github-com-cheggaaa-pb-v3
+                        go-github-com-kardianos-minwinsvc
+                        go-github-com-mattn-go-colorable
+                        go-github-com-mattn-go-isatty
+                        go-github-com-mattn-go-runewidth
+                        go-github-com-mitchellh-mapstructure
+                        go-github-com-olekukonko-tablewriter
+                        go-github-com-quic-go-quic-go
                         go-github-com-vividcortex-ewma
-                        go-github-com-arceliar-phony
-                        go-github-com-arceliar-ironwood)))))
+                        go-golang-org-x-crypto
+                        go-golang-org-x-net
+                        go-golang-org-x-text
+                        go-golang-org-x-tools
+                        go-golang-zx2c4-com-wireguard
+                        go-netlink
+                        go-netns)))))
     (home-page "https://yggdrasil-network.github.io/blog.html")
     (synopsis
      "Experiment in scalable routing as an encrypted IPv6 overlay network")
-- 
2.41.0

